.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- HOW TO USE: Under each "#### Heading" below, enter information relevant to your pull request.
+Comment blocks, surrounded with <!–– and ––>, won't be visible in the actual post.-->
+
+#### Summary
+
+SUMMARY: Category "Brief description"
```

Comment:
What if we get rid of the literal `SUMMARY:` and leave just `Category "Brief description"`?

Reply:
The fact that it's a summary is already conveyed by the section title, isn't it?